src/document/DocumentAnnotator.ts

```diff
@@ -30,6 +31,7 @@ export default class DocumentAnnotator extends BaseAnnotator {
         // Lazily instantiate managers as pages are added or re-rendered
         if (managers.size === 0) {
             managers.add(new RegionManager({ location: pageNumber, referenceEl: pageReferenceEl }));
+            managers.add(new HighlightManager({ location: pageNumber, referenceEl: pageReferenceEl }));
```

Comment:
It seems like text highlights should be rendered "under" regions, so they should probably be rendered first. What do you think?

Reply:
`The new layer element is inserted immediately after the reference element`. So `HighlightManager` should be constructed later in order to insert layer before region, then highlight can be rendered "under" regions.

Reply:
Hm, okay, that's my bad. We should consider refactoring `insert` to support a more intuitive order (first in, first rendered) for the managers. We can do that separately, though.

Reply:
Yeah I can open another pr later.